openslides/agenda/views.py

```diff
@@ -16,6 +17,8 @@
 from django.utils.translation import ugettext as _
 from django.utils.translation import ugettext_lazy
 from reportlab.platypus import Paragraph
+from rest_framework.decorators import list_route
```

Comment:
I think this is exaggerated. We import other things from other third party libs, too. 

But I can live with it, so: done